### PR TITLE
Add Java Bindings

### PR DESCRIPTION
**Pull request by KarthikRIyer**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright [yyyy] Contributors to the OpenTimelineIO project
```
Comment (reinecke) [inline LICENSE:189]: for `[yyyy]` you can put `2020`.

**Comment by KarthikRIyer** [comment]
I thought this was just the boilerplate that people could add to the files they contribute. The year could be anything, that's why I didn't change it.

**Comment by ssteinbach** [comment]
Would love to get input from @reinecke and others who have been tracking the java PR on the main repo to weigh in on anything that might need to be added to this repo.  Thanks!

File: LICENSE (hunk 1 above)
Comment (KarthikRIyer) [reply LICENSE:189]: Should I change the appendix to instruct people to use the SPDX format instead of what's given here?

**Pull request merged**